libs/types/src/auth/auth.ts

```diff
@@ -42,11 +42,52 @@ export const UserRoleSchema: z.ZodSchema<UserRole> = z.union([
   z.literal('cardless_voter'),
 ]);
 
+/**
+ * See libs/auth/src/lockout.ts for more context.
+ */
+export type NumIncorrectPinAttemptsAllowedBeforeCardLockout =
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10;
+export const NumIncorrectPinAttemptsAllowedBeforeCardLockoutSchema: z.ZodSchema<NumIncorrectPinAttemptsAllowedBeforeCardLockout> =
+  z.union([
+    z.literal(3),
+    z.literal(4),
+    z.literal(5),
+    z.literal(6),
+    z.literal(7),
+    z.literal(8),
+    z.literal(9),
+    z.literal(10),
+  ]);
+export const DEFAULT_NUM_INCORRECT_PIN_ATTEMPTS_ALLOWED_BEFORE_CARD_LOCKOUT: NumIncorrectPinAttemptsAllowedBeforeCardLockout = 5;
+
+/**
+ * See libs/auth/src/lockout.ts for more context.
+ */
+export type StartingCardLockoutDurationSeconds = 15 | 30 | 60;
+export const StartingCardLockoutDurationSecondsSchema: z.ZodSchema<StartingCardLockoutDurationSeconds> =
+  z.union([z.literal(15), z.literal(30), z.literal(60)]);
+export const DEFAULT_STARTING_CARD_LOCKOUT_DURATION_SECONDS: StartingCardLockoutDurationSeconds = 15;
+
 /**
  * The inactive/idle session time limit, after which the user must reauthenticate - a VVSG2
  * requirement
  */
-export type InactiveSessionTimeLimitMinutes = 5 | 10 | 15 | 20 | 30;
+export type InactiveSessionTimeLimitMinutes = 10 | 15 | 20 | 25 | 30;
```

Comment:
Thinking about our product some more, 5 minutes felt like too short of a time limit to even allow since operations like backups can frequently take longer than that